tests/drivers/entropy/api/entropy_psa_crypto.overlay

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/ {
+	chosen {
+		zephyr,entropy = &rng_psa;
+	};
+
+	rng_psa: entropy_psa_crypto {
+		compatible = "zephyr,psa-crypto-rng";
+		label = "entropy_psa_crypto";
```

Comment:
is this label used? otherwise remove it.

Reply:
Correct, it's not used. I'm remove it. Thanks ;)
PS = as a consequence the first comment is solved automatically